src/hooks/actions.tsx

```diff
@@ -446,13 +446,15 @@ export function useDisputeAction({
     errors: [prepareDisputePriceError?.message, disputePriceError?.message],
   };
 }
+
 export function useDisputeAssertionAction({
   query,
 }: {
   query?: OracleQueryUI;
 }): ActionState | undefined {
   const { disputeAssertionParams, chainId, actionType } = query ?? {};
   const { address } = useAccount();
+  console.log("HERE", disputeAssertionParams?.(address));
```

Comment:
you can remove this

Reply:
ooh i must've brought this over from another branch, good catch!